contracts/freeportParts/ERC20Adapter.sol

```diff
@@ -45,4 +45,9 @@ abstract contract ERC20Adapter is TransferOperator {
         currencyContract.transfer(user, amount);
     }
 
+    function sendShare(address beneficiar, uint256 amount) internal {
+        _burn(beneficiar, CURRENCY, amount);
```

Comment:
This should burn from the JA, not the benificiary.

Reply:
it is meant to burn amount of internal currency from JA addresses, right?